e2e/playwright/actions/src/tests/create-folder.spec.ts

```diff
@@ -23,145 +23,154 @@
+      await expect(folderDialog.createButton).toBeDisabled();
+      await expect(folderDialog.folderNameInputHint).toContainText(folderErrors.folderNameCantContainTheseCharacters);
     }
   });
 
   test('[C280406] Create a folder with a name containing only spaces', async ({ personalFiles }) => {
-    await personalFiles.acaHeader.createButton.click();
-    await personalFiles.matMenu.createFolder.click();
-    await personalFiles.folderDialog.folderNameInputLocator.fill('   ');
+    const folderDialog = personalFiles.folderDialog;
+
+    await personalFiles.selectCreateFolder();
+    await folderDialog.folderNameInputLocator.fill('   ');
 
-    await expect(personalFiles.folderDialog.createButton).toBeDisabled();
-    await expect(personalFiles.folderDialog.folderNameInputHint).toContainText(folderErrors.folderNameCantContainOnlySpaces);
+    await expect(folderDialog.createButton).toBeDisabled();
+    await expect(folderDialog.folderNameInputHint).toContainText(folderErrors.folderNameCantContainOnlySpaces);
   });
 
   test('[C216349] Cancel folder creation', async ({ personalFiles }) => {
-    await personalFiles.acaHeader.createButton.click();
-    await personalFiles.matMenu.createFolder.click();
+    const folderDialog = personalFiles.folderDialog;
+
+    await personalFiles.selectCreateFolder();
     await expect(personalFiles.page.getByRole('dialog', { name: 'Create new folder' })).toBeVisible();
-    await personalFiles.folderDialog.folderNameInputLocator.fill(randomFolderName);
-    await personalFiles.folderDialog.cancelButton.click();
+    await folderDialog.folderNameInputLocator.fill(randomFolderName);
+    await folderDialog.cancelButton.click();
     await expect(personalFiles.page.getByRole('dialog', { name: 'Create new folder' })).toBeHidden();
   });
 
   test('[C216350] Duplicate folder name error', async ({ personalFiles }) => {
-    await personalFiles.acaHeader.createButton.click();
-    await personalFiles.matMenu.createFolder.click();
-    await personalFiles.folderDialog.folderNameInputLocator.fill(randomFolderName);
-    await personalFiles.folderDialog.createButton.click();
-    // duplicate folder name
-    await personalFiles.acaHeader.createButton.click();
-    await personalFiles.matMenu.createFolder.click();
-    await personalFiles.folderDialog.folderNameInputLocator.fill(randomFolderName);
-    await personalFiles.folderDialog.createButton.click();
+    const folderDialog = personalFiles.folderDialog;
+    const folderSnackBar = personalFiles.snackBar;
 
-    await expect(personalFiles.page.getByRole('dialog', { name: 'Create new folder' })).toBeVisible();
-    await expect(personalFiles.snackBar.getByMessageLocator(folderErrors.thereIsAlreadyAFolderWithThisName)).toBeVisible();
+    await personalFiles.selectCreateFolder();
+    await folderDialog.createNewFolderDialog(commonFolderName, null, null);
```

Comment:
if  createNewFolderDialog method has optional parameter you don't need to send null